### DISCUSSION TITLE
<USERNAME> tag required in regex in fail2ban setup docs

### DISCUSSION THREAD

**Discussion by bluppfisk** [dani-garcia/vaultwarden#4667] [General]
should be updated to contain the USERNAME tag otherwise fail2ban complains about there not being a failure-id group in Username.

(That said, my fail2ban isn't banning although it is recognising the IP address now)

```
# path_f2b/filter.d/vaultwarden.local

[INCLUDES]
before = common.conf

[Definition]
failregex = ^.*?Username or password is incorrect\. Try again\. IP: <ADDR>\. Username: <USERNAME>.*$
ignoreregex =
```

**Comment by BlackDex** [comment]
Go ahead. It's a wiki which can be maintained by the community 😄
